src/main/java/bsh/Interpreter.java

```diff
@@ -525,6 +527,12 @@ public void run()
                 if(!interactive)
                     eof = true;
             }
+            catch (TokenMgrException e) {
+                if ( DEBUG.get() )
+                    e.printStackTrace();
+                if(!interactive)
+                    eof = true;
+            }
             catch(Exception e)
```

Comment:
You were still explaining to me why we need TokenMgrException now, I forget.

Reply:
Sorry, it's a leftover. I will remove it.